### PR TITLE
Nico mongodb

### PR DESCRIPTION
**Pull request by qleseid**

### CODE REVIEW

File: backend/routes/user.routes.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export const user3Routes = express.Router();
+
+user3Routes.post('/register', async (req: any, res: any) => {
+  try {
+    const user = new User(req.body);
+    await user.save();
+    const token = await user.generateAuthToken();
+    res.status(201).send({user, token});
+  } catch (error) {
+    res.status(400).send(error);
```
Comment (closs290) [inline backend/routes/user.routes.ts:14]: Is this really a bad request? Or is this a 500 server-side error?

File: backend/routes/user.routes.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import express = require('express');
+import { User } from '../models/user.models';
+import { auth } from '../middleware/auth';
+
+export const user3Routes = express.Router();
```
Comment (closs290) [inline backend/routes/user.routes.ts:5]: I think this is Lucas' code, not yours, but I'm curious about the magic '3' here. Should we be embedding 'magic numbers' (a set number of possible routes, for example) into our code?

**Review by closs290 — APPROVED** [review]
Thanks for doing this heavy lifting!

File: backend/routes/user.routes.ts (hunk 2 above)
Comment (nicholasjuncos) [reply backend/routes/user.routes.ts:5]: That has been changed. I wrote 3 different variations of routes and functions, deleted 2 and forgot to change the final product variable name. Nothing more than that.

File: backend/routes/user.routes.ts (hunk 1 above)
Comment (nicholasjuncos) [reply backend/routes/user.routes.ts:14]: 400 is a bad-request error.

**Pull request merged**